### ISSUE TITLE
Include label in the metadata

### ISSUE THREAD

**Issue by wibed** [apple/swift-log#336]
is there interest in including label in the metadata by default.

likely to the metadataprovider, including label per default within the metadata would have any logger specific information
within the handler? maybe even include log level aswell.

i would provide a pr

**Comment by FranzBusch** [comment]
I don't think this is the right thing to do for this API package. The various logging backends can decide to include the label and the log level by default if they want to.
